src/styles/style.css

```diff
@@ -0,0 +1,209 @@
+  margin-left: calc(100px * 1 + 1px * 1 + (10px) + (1px) * 2);
+}
+
+.calendar--start-day--wed > .calendar__day:first-child {
+  margin-left: calc(100px * 2 + 1px * 2 + (10px) + (1px) * 2);
+}
+
+.calendar--start-day--thu > .calendar__day:first-child {
+  margin-left: calc(100px * 3 + 1px * 3 + (10px) + (1px) * 2);
+}
+
+.calendar--start-day--fri > .calendar__day:first-child {
+  margin-left: calc(100px * 4 + 1px * 4 + (10px) + (1px) * 2);
+}
+
+.calendar--start-day--sat > .calendar__day:first-child {
+  margin-left: calc(100px * 5 + 1px * 5 + (10px) + (1px) * 2);
+}
+
+.calendar--start-day--sun > .calendar__day:first-child {
+  margin-left: calc(100px * 6 + 1px * 6 + (10px) + (1px) * 2);
+}
+
+.calendar--month-length--31 > .calendar__day:nth-child(n + 32) {
+  display: none;
+}
+
+.calendar--month-length--30 > .calendar__day:nth-child(n + 31) {
+  display: none;
+}
+
+.calendar--month-length--29 > .calendar__day:nth-child(n + 30) {
+  display: none;
+}
+
+.calendar--month-length--28 > .calendar__day:nth-child(n + 29) {
+  display: none;
+}
+
+.calendar__day:nth-child(1)::before {
+  content: '1';
+}
+
+.calendar__day:nth-child(2)::before {
+  content: '2';
+}
+
+.calendar__day:nth-child(3)::before {
+  content: '3';
+}
+
+.calendar__day:nth-child(4)::before {
+  content: '4';
+}
+
+.calendar__day:nth-child(5)::before {
+  content: '5';
+}
+
+.calendar__day:nth-child(6)::before {
+  content: '6';
+}
+
+.calendar__day:nth-child(7)::before {
+  content: '7';
+}
+
+.calendar__day:nth-child(8)::before {
+  content: '8';
+}
+
+.calendar__day:nth-child(9)::before {
+  content: '9';
+}
+
+.calendar__day:nth-child(10)::before {
+  content: '10';
+}
+
+.calendar__day:nth-child(11)::before {
+  content: '11';
+}
+
+.calendar__day:nth-child(12)::before {
+  content: '12';
+}
+
+.calendar__day:nth-child(13)::before {
+  content: '13';
+}
+
+.calendar__day:nth-child(14)::before {
+  content: '14';
+}
+
+.calendar__day:nth-child(15)::before {
+  content: '15';
+}
+
+.calendar__day:nth-child(16)::before {
+  content: '16';
+}
+
+.calendar__day:nth-child(17)::before {
+  content: '17';
+}
+
+.calendar__day:nth-child(18)::before {
+  content: '18';
+}
+
+.calendar__day:nth-child(19)::before {
+  content: '19';
+}
+
+.calendar__day:nth-child(20)::before {
+  content: '20';
+}
+
+.calendar__day:nth-child(21)::before {
+  content: '21';
+}
+
+.calendar__day:nth-child(22)::before {
+  content: '22';
+}
+
+.calendar__day:nth-child(23)::before {
+  content: '23';
+}
+
+.calendar__day:nth-child(24)::before {
+  content: '24';
+}
+
+.calendar__day:nth-child(25)::before {
+  content: '25';
+}
+
+.calendar__day:nth-child(26)::before {
+  content: '26';
+}
+
+.calendar__day:nth-child(27)::before {
+  content: '27';
+}
+
+.calendar__day:nth-child(28)::before {
+  content: '28';
+}
+
+.calendar__day:nth-child(29)::before {
+  content: '29';
+}
+
+.calendar__day:nth-child(30)::before {
+  content: '30';
+}
+
+.calendar__day:nth-child(31)::before {
+  content: '31';
+}
```

Comment:
The repetitive code for setting the content of each day could be optimized using a CSS preprocessor like SCSS. Consider using a loop to generate these styles, as seen in your SCSS files.